gateleen-kafka/src/main/resources/gateleen_kafka_topic_configuration_schema

```diff
@@ -0,0 +1,35 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": {
+    "$ref": "#/definitions/Topic"
+  },
+  "definitions": {
+    "Topic": {
+      "description": "A single kafka topic entry",
+      "type": "object",
+      "additionalProperties": {
+        "type": "string"
+      },
+      "properties": {
+        "bootstrap.servers": {
```

Comment:
Do we really want _dot_ in property name? IMHO a candidate to provocate confusion.

Reply:
Yes we do, because these are Kafka related properties which must be called exactly like defined in the schema. See [Kafka doc](https://kafka.apache.org/documentation/#producerconfigs)